channels/mx.m3u

```diff
@@ -1,71 +1,75 @@
 #EXTINF:-1 tvg-id="XHBJTDT.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/en/0/0e/Canal45Tijuana.png" group-title="Local",Canal 45 (XHBJ-TDT) (1080p) [Offline]
 https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhbj-tdt.m3u8
-#EXTINF:-1 tvg-id="Canal5.mx" tvg-country="MX" tvg-language="" tvg-logo="https://i.imgur.com/mw4992J.png" group-title="Local",Canal 5 (XETV-TDT) (1080p) [Offline]
+#EXTINF:-1 tvg-id="Canal5.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/mw4992J.png" group-title="Local",Canal 5 (XETV-TDT) (1080p) [Offline]
 https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xetv-tdt.m3u8
 #EXTINF:-1 tvg-id="CanalCatorce.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/canalcatorcemx/picture?width=200&height=200" group-title="Local",Canal Catorce (720p)
 https://d3nljkrx6mjqra.cloudfront.net/out/v1/1b9d9efd27814b3b8dc570113ae54409/index.m3u8
-#EXTINF:-1 tvg-id="CanaldelasEstrellas.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="" group-title="Local",Canal de Las Estrellas
+#EXTINF:-1 tvg-id="CanaldelasEstrellas.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://www.lasestrellas.tv/assets/08680ecc38e44f53/images/icn_lasestrellas_512x512.png" group-title="Local",Canal de Las Estrellas (480p)
 http://38.131.11.9:1080/play/a032
 #EXTINF:-1 tvg-id="CanalMundoPlus.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/9BtSyV2.png" group-title="",Canal Mundo+ (560p)
 http://vcp1.myplaytv.com:1935/mundomas/mundomas/playlist.m3u8
-#EXTINF:-1 tvg-id="Capital21.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="" group-title="Local",Capital 21 [Not 24/7]
+#EXTINF:-1 tvg-id="Capital21.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/P1pHy3j.jpg" group-title="Local",Capital 21 [Not 24/7]
 https://video.cdmx.gob.mx/livestream/stream.m3u8
-#EXTINF:-1 tvg-id="ConectaTV.mx" tvg-country="MX" tvg-language="" tvg-logo="https://i.imgur.com/9TGQZzk.png" group-title="Music",Conecta TV (720p)
+#EXTINF:-1 tvg-id="ConectaTV.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/9TGQZzk.png" group-title="Music",Conecta TV (720p)
 http://204.12.211.210:1935/conectatv/conectatv/playlist.m3u8
-#EXTINF:-1 tvg-id="ConectaTV.mx" tvg-country="MX" tvg-language="" tvg-logo="https://i.imgur.com/9TGQZzk.png" group-title="Music",Conecta TV (720p)
+#EXTINF:-1 tvg-id="ConectaTV.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/9TGQZzk.png" group-title="Music",Conecta TV (720p)
 #EXTVLCOPT:http-referrer=https://ssh101.com/
 https://tna5.bozztv.com/Conectatvmexico/index.m3u8
+#EXTINF:-1 tvg-id="Forotv.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/uNQVYem.png" group-title="News",ForoTV (720p)
+https://live-streams-notusa.televisa.com/channel02-b/index.m3u8
```

Comment:
That one sounds like a backup link to https://live-streams-notusa.televisa.com/channel02/index.m3u8

Reply:
maybe  this one pull out from site